### PR TITLE
Added deploy metrics to deploy csv report

### PR DESCRIPTION
**Pull request by stephennyu**
1. Moved logic to fetch cycle_time into deploy model
2. Added cycle_time into Deploy CSV

/cc @zendesk/samson

### CODE REVIEW

File: app/models/deploy.rb [hunk 1]
```diff
@@ -135,6 +135,10 @@ def duration
     updated_at - start_time
   end
 
+  def cycle_time
+    @cycle_time ||= DeployMetrics.new(self).cycle_time
```
Comment (grosser) [inline app/models/deploy.rb]: remove the nil logic from DeployMetrics then

**Comment by grosser** [comment]
is this needed multiple times ... can this be a 1-off export instead ?

**Comment by stephennyu** [comment]
1. Moved this to a separate presenter
2. Converted it to a one-off export (export whole projects at once)

File: app/presenters/deploy_metrics_csv_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'csv'
+
+class DeployMetricsCsvPresenter
```
Comment (grosser) [inline app/presenters/deploy_metrics_csv_presenter.rb]: stupid-simple ❤️

**Pull request merged**